assignments/callbacks.js

```diff
@@ -27,29 +27,77 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length)
 }
 
+getLength(items, function(length) {
+  console.log("Q1 Callbacks: " + length);
+})
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.length - 1);
 }
 
+last(items, function(lastItem) {
+  console.log("Q2 Callbacks: " + lastItem);
```

Comment:
Read the question again

Reply:
It says "last passes the last item of the array into the callback". 

The code below passes the last item of the array into the callback:

```return cb(**arr.length - 1**);```

Where have I misunderstood that?